SU-SWS/stanford_profile
# D8CORE-1290: setting the width so a single banner fills the parent

# READY FOR REVIEW
- I need to test this more. 

# Summary
- D8CORE-1290. A single banner was not filling the parent container if it is the only object on the page. 
- I was also able to stack images (sm/med) and have them align center, left and right. This was a noted problem by @pookmish  in the comments on this ticket. 

# Needed By (Date)
- Thursday

# Urgency
- Medium

# Steps to Test

1. Pull in this change.
2. Build.
3. Create a page with no layout and only one banner. 
4. Notice the banner fills the container. 
5. To test the other issue, add a text container with sm/med images that are aligned left, center and right. 
6. Notice they are aligned as you wanted and not a single left aligned column.
7. Try interior pages to confirm nothing funny happened with the header or the navigation bar. 
8. Make sure the header is left aligned with the column of text.

# Affected Projects or Products
- D8CORE-1290
- Profiles since it was on the Stanford page. 


# Associated Issues and/or People
- D8CORE-1290
- @pookmish's comment in the ticket
- @kerri-augenstein - Confirmed this morning that she liked the 100% size of the banner image.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
